test-ui-test/ACTUAL.TXT

```diff
@@ -0,0 +1,31 @@
+    ____________________________________________________________
```

Comment:
Given that ACTUAL.TXT is always recreated when you run the test, I don't see any reason to commit it